src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt

```diff
@@ -119,6 +119,10 @@ object DiscordRPC : Module() {
             LineInfo.DIMENSION -> {
                 InfoCalculator.dimension()
             }
+            LineInfo.TASK -> {
```

Comment:
TASK is vague maybe make it something more related to the highway module. Maybe HIGHWAY or something.

Reply:
Ah, Highway Work